public/js/clients/firefox/commands.js

```diff
@@ -46,9 +46,12 @@ function sourceContents(sourceId) {
 
 function setBreakpoint(location, condition) {
   const sourceClient = threadClient.source({ actor: location.sourceId });
+
+  const column = location.column === 0 ? undefined : location.column;
```

Comment:
I don't want to do this here. If you pass a column of 0, it should send a column of 0. Firefox's behavior makes sense: if it's a real value then it should only set a breakpoint on that column.

You should move this logic up to source maps and decipher the column info there.